README.md

```diff
@@ -1,8 +1,35 @@
-# React + Vite
+- final project backend submission
+# newsexplorerfinal project
 
```

Comment:
You need to **format** the markup (and code) to show **indentation** and to make the code more readable. You can do that with `Prettier` or a combination: press `SHIFT + ALT + F`. **Please, correct it everywhere in the code**


At least, you should format the code in `Footer`